packages/hoppscotch-common/src/helpers/import-export/import/workers/openapi-import-worker.ts

```diff
@@ -0,0 +1,53 @@
+import { Buffer } from "buffer"
+import process from "process"
+
+// Set up global shims for the swagger-parser library
+self.Buffer = Buffer
+self.global = self
+self.process = process
```

Comment:
A runtime exception arises while spinning up the production build about `process` not defined in the browser environment. Could you take a look?

Reply:
sent a fix.